css/section-2.css

```diff
@@ -0,0 +1,28 @@
+.who {
+  color: #b35efe;
+  font-family: "Open Sans", sans-serif;
+  font-size: 1rem;
+  font-weight: bolder;
+}
+#about-part-2 h1 {
+  color: #292c34;
+  font-family: "Open Sans", sans-serif;
+  font-size: 4rem;
+  margin-block-start: 1rem;
+  margin-block-end: 1rem;
+}
+
+#about-part-2 h1::after {
+  content: "";
+  width: 100%;
+  background-color: #b35efe;
```

Comment:
Change it to 
`background-color: var(--color-denim);`
`background-color: var(--color-denim--dark);`

Like some color matching with the color scheme, as you can see in `main.css`

Reply:
😀

Reply:
sir After changing colour its look like this so i dont change.😇
<img width="204" alt="Screenshot 2021-06-20 at 4 42 32 PM" src="https://user-images.githubusercontent.com/77855395/122671950-a53f3b80-d1e6-11eb-992f-2b3b55fbc4d3.png">